Warning upon listing below floor

magiclars-off opened the pull request:
Warns the user upon creating a listing more than 10% below floorprice.

![image](https://user-images.githubusercontent.com/62888804/151367431-678ee16d-eeb5-4f9b-92f9-46539eecb21b.png)



Also fixes a small bug where parseFloat was fed a NaN after clearing the input 

![image](https://user-images.githubusercontent.com/62888804/151368037-e84123a8-c9d3-4452-b672-3d293555176e.png)


@wyze @jcheese1

wyze commented on src/pages/inventory/[[...section]].tsx:
```diff
@@ -516,6 +545,9 @@ const Drawer = ({
                             })}
                           </div>
+                          {floorDifference > 10 && (
+                            <p className="flex justify-center text-red-700 font-bold">{`WARNING: ${floorDifference}% below floorpice`}</p>
```
```suggestion
                            <p className="flex justify-center text-red-700 font-bold">{`WARNING: ${floorDifference}% below floor price`}</p>
```
Typo here, and added a space between the words.

wyze reviewed (changes requested):
Can we also maybe change the background color or the button for a little more visual indication that they should pay attention? Maybe there is a good shade of yellow that wouldn't look terrible. Since our theme is red.

magiclars-off commented:
Thank you for the feedback, you are right. The warning didn't fit well in the UI

I made some changes

- moved the warning under the price input
- moved the royalties / share component down above the button and increased the font-size a little bit for readability

![image](https://user-images.githubusercontent.com/62888804/151454229-5ca4076f-0f5e-43b5-a725-634c8094ac82.png)

wyze commented:
@magiclars-off Looks good, except can we clear the error if I remove the value from the input field?

magiclars-off commented:
@wyze done

wyze reviewed (approved):
Awesome, thank you!
